README.md

```diff
@@ -99,4 +99,4 @@ Function[Symbol.operator('::')] = function (lho, rho) {
 
 ### How could this even work?
 
-When parsing a piece of javascript, when the tokenizer reached an unrecognised token between one or two operands (with, perhaps a whitelist of certain allowed operator characters, for example any Unicode character betwixt U+2000–U+218F), it could search in the operator registry (perhaps at runtime? during evaluation?) for the operator, and if not found could raise a SyntaxError.
\ No newline at end of file
+When parsing a piece of javascript, when the tokenizer reached an unrecognised token between one or two operands (with, perhaps a whitelist of certain allowed operator characters, for example any Unicode character betwixt U+2000–U+218F), it could search in the operator registry (perhaps at runtime? during evaluation?) for the operator, and if not found could raise a SyntaxError.
```

Comment:
I didn't change anything on this line, not sure why GitHub is showing it as changed.

Reply:
Perhaps your text editor added a previously-missing \n at the end of the file.

Reply:
I made this change using Github's editor. :(